File: Kotlin/BibliotecaKT/Llibreclass.kt
package Biblio

class Llibre(val titol: String, val autor: String, var exemplarsDisponibles: Int) {
    fun prestar() {
        if (exemplarsDisponibles > 0) {
            exemplarsDisponibles--
            println("El llibre $titol de l'autor $autor s'ha prestat.")
        } else {
            println("No hi ha exemplars disponibles del llibre $titol.")
        }
    }

    fun devolver() {
        exemplarsDisponibles++
        println("El llibre $titol de l'autor $autor s'ha retornat.")
    }

    fun mostrarInformacio() {
        println("Títol: $titol, Autor: $autor, Exemplars disponibles: $exemplarsDisponibles")
    }
} 

File: BibliotecaPY/Llibreclass.py
class Llibre:
    def __init__(self, titol, autor, quantitat):
        self.titol = titol
        self.autor = autor
        self.quantitat = quantitat

    def prestar(self):
        if self.quantitat > 0:
            self.quantitat -= 1
            print(f"Prestant el llibre: {self.titol} de l'autor: {self.autor}")
        else:
            print(f"No queden exemplars del llibre: {self.titol}")

    def retornar(self):
        self.quantitat += 1
        print(f"Retornant el llibre: {self.titol} de l'autor: {self.autor}")

    def mostrar_info(self):
        print(f"Títol: {self.titol}, Autor: {self.autor}, Quantitat disponible: {self.quantitat}")
